lib/async_hooks.js

```diff
@@ -71,8 +71,7 @@ function fatalError(e) {
     Error.captureStackTrace(o, fatalError);
     process._rawDebug(o.stack);
   }
-  if (process.execArgv.some(
-      (e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
+  if (process.execArgv.some((e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
```

Comment:
As of May, the linter doesn't enforce the line-length rule on lines with regular expression literals. See https://github.com/nodejs/node/pull/12807 for rationale. (Working as intended here too, IMO. I find this much more readable than either the current formatting or any other reasonable formatting I was able to come up with.)